contrib/format-excel/src/test/java/org/apache/drill/exec/store/excel/TestExcelFormat.java

```diff
@@ -339,11 +339,12 @@ public void testInconsistentDataQuery() throws Exception {
 
     testBuilder()
       .sqlQuery(sql)
-      .ordered().baselineColumns("col1", "col2")
-      .baselineValues("1.0", "Bob")
-      .baselineValues("2.0", "Steve")
-      .baselineValues("3.0", "Anne")
-      .baselineValues("Bob", "3.0")
+      .ordered()
```

Comment:
Your test does not include order by thus returned result order is unpredictable. Please change to unordered here and in other tests above.

Reply:
Fixed